gradle.properties

```diff
@@ -1,3 +1,2 @@
 org.gradle.caching=true
 org.gradle.parallel=true
-org.gradle.vfs.watch=true
```

Comment:
I was wondering why, found it:
https://docs.gradle.org/7.0/release-notes.html#file-system-watching

Reply:
Yep, it's the default.